python/tests/server/test_worker.py

```diff
@@ -104,14 +115,17 @@ class Result:
     exception: Optional[Exception] = None
 
 
-def _process(events, swallow_exceptions=False):
+async def _process(events) -> Result:
+    return _sync_process([e async for e in events])
+
+
+def _sync_process(events) -> Result:
```

Comment:
Nit and not a blocker: it would be more conventional for the async function to have the name with an `a` or an `async` in.

Reply:
the sync variant is used in one place, the async version is used everywhere else, it seems silly